app/src/main/java/edu/calvin/cs262/teamc/activity_PetforAdoption.java

```diff
@@ -168,28 +156,22 @@ public void saveNewPetSwipe (View view) {
         TextView ImageURL = findViewById(R.id.chooseFile);
 
         if (TextUtils.isEmpty(NameText.getText())) {
+
             NameText.setError(getString(R.string.Name_Required));
+
```

Comment:
Why these extra lines here (159, 161, 163)?

Reply:
Not sure exactly which lines you are referring to (cause I have deleted so many lines in the file so far), but I just checked around here and I'm not seeing any major problems.